frontend/javascripts/test/puppeteer/dataset_rendering.screenshot.ts

```diff
@@ -114,7 +114,7 @@ test.before("Retrieve dataset ids", async () => {
       3,
       async () => {
         const options = getDefaultRequestOptions(URL);
-        const url = `${URL}/api/datasets/disambiguate/sample_organization/${datasetName}/toId`;
+        const url = `${URL}api/datasets/disambiguate/sample_organization/${datasetName}/toId`;
```

Comment:
the rest of the screenshot tests use `urljoin` to join url parts. that way, double slashes can also be avoided. I'd rather do this then relying on a trailing slash.

Reply:
Oh I overlooked that. Thanks for the suggestion. It should be applied now. 

I also did a quick test that requesting the URLs still works